### DISCUSSION TITLE
BTicino BT-F40T125A 3Phase detected as "Not supported

### DISCUSSION THREAD

**Discussion by serbat** [Koenkk/zigbee2mqtt#21929] [Devices]
Hi everyone
I purchased the Bticino F40T125A meter and only later realized that it is not officially supported, I wonder is it possible to support it, has anyone ever done it?

Grazie..

![immagine](https://github.com/Koenkk/zigbee2mqtt/assets/83161647/bb3328d1-7dcc-4adc-9445-f38706f41604)

**Comment by serbat** [comment]
Image

![432783600_769314481806895_5796107869097209288_n](https://github.com/Koenkk/zigbee2mqtt/assets/83161647/6673e9ee-f68a-435d-8a05-dcf7c4c1b897)

**Comment by mtshare** [comment]
same here. Any news?
